snap/info.go

```diff
@@ -402,6 +403,30 @@ func (app *AppInfo) SecurityTag() string {
 	return AppSecurityTag(app.Snap.Name(), app.Name)
 }
 
+func (app *AppInfo) DesktopFiles() []string {
+	dfs, err := filepath.Glob(filepath.Join(dirs.SnapDesktopFilesDir, fmt.Sprintf("%s_*.desktop", app.Snap.Name())))
+	if err != nil {
+		return nil
+	}
+	appName := filepath.Base(app.WrapperPath())
+	re, err := regexp.Compile(fmt.Sprintf(`(?m)^Exec=%s .*`, appName))
```

Comment:
I thought we had a strong convention for how to name desktop files after the app name?